regression/tests/studio/test_course_pages.py

```diff
@@ -123,22 +121,29 @@ def setUp(self):
         lms_login = LmsLoginApi()
         lms_login.authenticate(self.browser)
 
+        self.course_page = CourseInfoPageExtended(
+            self.browser, get_course_key(self.course_info)
+        )
+        self.courseware_page = CoursewarePageExtended(
+            self.browser,
+            get_course_key(self.course_info)
+        )
         self.pages_page = PagesPageExtended(
             self.browser,
             self.course_info['org'],
             self.course_info['number'],
             self.course_info['run']
         )
+        # Login to LMS to avoid authentication issue
+        self.course_page.visit()
         self.pages_page.visit()
 
     def test_view_live_pages(self):
```

Comment:
Please remove this testcase.
If the functionality that it tests is broken, then we will be alerted by the assert_page_is_shown_in_lms test which starts off by doing the same thing.